README.md

```diff
@@ -29,15 +29,29 @@ Also:
 
 Like others, Smithay as a compositor library has its own sample compositor: anvil.
 
-You can run it with cargo after having cloned this repository:
-
+You can run it with cargo after having cloned this repository and installed the required dependencies:
+
+- `libdbus-glib-1`
+- `libudev`
+- `libsystemd`
```

Comment:
Not sure that systemd would apply to every udev session type?

Reply:
It didn't compile without it for me, maybe it's picking it up from my environment somehow?

Are there other libraries that might be needed instead for non-systemd distros?

Reply:
I don't know either, I'd probably need to spin up a bunch of vms to test this

Reply:
The system dependencies would depend on which cargo features are enabled, `libsystemd` is pulled by `backend_session_logind`.

Reply:
Same way [seatd/libseat](https://sr.ht/~kennylevinsen/seatd/), requires a libseat system lib as well